vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -121,9 +121,9 @@ class LEDGER_EXPORT Ledger {
   virtual std::string URIEncode(const std::string& value) = 0;
 
   virtual void SetPublisherInfo(std::unique_ptr<PublisherInfo> publisher_info,
```

Comment:
I don't understand why any of this needs window_id?

Reply:
removed as this was an left over that we don't use anymore